sopel/modules/reddit.py

```diff
@@ -162,9 +150,9 @@ def redditor_info(bot, trigger, match=None):
 
 
 # If you change the groups here, you'll have to change some things above.
-@rule('.*%s.*' % user_url)
-def auto_redditor_info(bot, trigger):
-    redditor_info(bot, trigger)
+@url(user_url)
+def auto_redditor_info(bot, trigger, match=None):
```

Comment:
The `match` parameter should be always provided when `@url` is used instead of `@rule`.

```suggestion
def auto_redditor_info(bot, trigger, match):
```